src/LykkePublicAPI/Controllers/CandlesController.cs

```diff
@@ -42,8 +74,9 @@ public CandlesController(ICandlesHistoryServiceProvider candlesServiceProvider)
         /// <param name="assetPairId">Asset pair Id</param>
         /// <param name="market">Market type</param>
         /// <param name="request">Request model</param>
+        [Obsolete("This method will be removed in future versions. Please, migrate your implementation on using the [GET] alternative method.")]
         [HttpPost("history/{assetPairId}/{market?}")]
-        [ProducesResponseType(typeof(CandlesHistoryResponse), 200)]
+        [ProducesResponseType(typeof(CandlesHistoryResponse<ApiCandle>), 200)]
```

Comment:
It's better to leave obsolete method signature without any changes at all.

Reply:
In general, no argues from my side. But here we have no client DLL, and we only handle HTTP-requests, providing JSON responses. There is no difference in JSON between CandlesHistoryResponse<ApiCandle> and the older signature - this was the main reason for me to modify it.